app/merginapi.cpp

```diff
@@ -551,6 +626,22 @@ int MerginApi::diskUsage() const
   return mDiskUsage;
 }
 
+void MerginApi::pingMergin()
+{
+  if ( mApiVersionStatus == MerginApiStatus::PASSED ) return;
+
+  setApiVersionStatus( MerginApiStatus::UNKNOWN );
+  emit apiIncompatibilityOccured( QStringLiteral( "Reset Mergin version" ), QStringLiteral( "pingMergin request about to send" ) );
```

Comment:
why do we emit the signal here?

Reply:
To avoid having blank loading while waiting for ping. Maybe we can have different notification message or different graphic while input sends ping request.
<img width="632" alt="Screenshot 2019-05-20 at 16 40 56" src="https://user-images.githubusercontent.com/6735606/58030047-26b2d400-7b1e-11e9-93a9-1635b9bd989c.png">